routes/web.php

```diff
@@ -20,5 +21,8 @@
 Route::get('/about', [MainController::class, 'aboutUs'])
     ->name('home.about');
 
-Route::get('/contact', [MainController::class, 'contact'])
-    ->name('home.contact');
+Route::get('/contact', [ReportController::class, 'show'])
+    ->name('report');
```

Comment:
report - это скорее про отчет
а у нас про форму обратной связи
```suggestion
    ->name('contact');
```
ниже тоже поправь report.store -> contact.store

не забудь потом в шаблонах поправить